src/nodedef.h

```diff
@@ -668,6 +668,14 @@ class NodeDefManager {
 	 */
 	void addNameIdMapping(content_t i, std::string name);
 
+	/*!
+	 * Removes a content ID from all groups.
+	 * Erases content IDs from vectors in \ref m_group_to_items and
+	 * removes empty vectors.
+	 * @param i a content ID
```

Comment:
Self-explanatory and incorrect comment. Please correct to `@param id Content ID` or remove.